docs/general/extensions.md

```diff
@@ -55,6 +55,20 @@ This script removes Google's tracking feature from the links in Google search re
 
 Its source code is [available on GitHub](https://github.com/Rob--W/dont-track-me-google). This userscript can be downloaded from [GreasyFork](https://greasyfork.org/en/scripts/428243-don-t-track-me-google) and installed in any AdGuard CoreLibs-based app.
 
+#### SponsorBlock
+
+From v7.22, AdGuard for Windows is capable of running scripts in SPA websites, such as YouTube. This enhancement also allows us to confidently recommend the SponsorBlock userscript.
```

Comment:
Let's not mention the exact version.

All desktop AdGuard versions (Win, macOS) will support it